plugins/CoreHome/angularjs/notification/notification.controller.js

```diff
@@ -20,6 +20,7 @@
                 return;
             }
 
+            piwikApi.withTokenInUrl();
```

Comment:
> 👍 we'd just need to double check that we also send the token in the actual request from the notification JS otherwise it won't work basically (but it's not working there currently anyway so could also ignore it and not test it)

@tsteur this pr contains this modification here ⬆️